packages/openapi3/src/lib.ts

```diff
@@ -104,6 +115,16 @@ const EmitterOptionsSchema: JSONSchemaType<OpenAPI3EmitterOptions> = {
         "  - `openapi.Org1.Service2.v1.1.yaml`    ",
       ].join("\n"),
     },
+    "openapi-versions": {
+      type: "array",
```

Comment:
What if this array is provided, but empty, what will we do in that case?

Reply:
And what if the same version variant is provided twice, e.g. `["3.1.0", "3.1.0"]`

Reply:
I added more constraints. If the array is provided and empty, an error will be thrown saying at least 1 item needs to be specified.

If the array is not provided, we use the default `[3.0.0]`.

Versions specified more than once before didn't cause any functional problems with the output, but now will warn that each element of the array needs to be unique.